### PR TITLE
Space-X Project

### PR DESCRIPTION
**Pull request by Sajeelzafar**
## In this project, the following milestones have been achieved:

- Set up a Navbar to navigate to Rockets, Missions, and profile pages using routers.
- Setting up redux components in ducks pattern.
- Fetching the data from the API.
- Storing the data in Redux store, state.
- Displaying data using React Components on the page.
- Adding reservation to the state.
- Create an action creator to toggle reservations in the state when the button is pressed.
- Using React conditional syntax, changing the styling and text of the button when pressed.
- Display the reserved rockets and joined missions on my profile page using the filter() method.
- Create tests for all the components.
- Deployed project using netlify.
- Create professional README.md, including a live demo link.
- Use React Bootstrap for styling.

### CODE REVIEW

File: src/redux/missions/missions.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// import axios from 'axios';
```
Comment (rloterh) [inline src/redux/missions/missions.js:2]: - [ ] OPTIONAL
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

File: src/components/pages/myprofile/Profile.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import {
+  Container, Row, Col,
+} from 'react-bootstrap';
+// import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (rloterh) [inline src/components/pages/myprofile/Profile.js:7]: - [ ] OPTIONAL
There are commented code blocks found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

File: README.md [hunk 3]
```diff
@@ -1 +1,165 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (rloterh) [inline README.md:165]: - [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide template instructions below it.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Ibrahim & Sajeel,  

Your project is complete, nice job 🥇 There is nothing else to say other than... it's time to merge it 🍾🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Implemented required components
- State management with redux is properly done
- Data is properly fetched from the endpoint via store
 - Navigations work as intended
 - Reviewed projects in turns
- Good unit test coverage

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (MohamedCK) [reply README.md:165]: I will work on that thanks for the reminder @rloterh

File: src/components/pages/myprofile/Profile.js (hunk 2 above)
Comment (MohamedCK) [reply src/components/pages/myprofile/Profile.js:7]: Thank you I will remove any commented code. 😄

**Pull request merged**